src/Http/Controllers/U2fController.php

```diff
@@ -35,90 +29,83 @@ public function __construct(LaravelU2f $u2f, Config $config)
     }
 
     /**
-     * @author LAHAXE Arnaud
```

Comment:
Please add him back, and add yourself if you want. He wrote most of this repo :)